index.css

```diff
@@ -204,35 +202,28 @@ section ul, section li {
 */
 
 .proposal-id {
-    color: #888;
+    color: #999;
```

Comment:
We should keep this at `#888`, rather than introducing slight variations in greys.